src/CodeGeneration/ApiGenerator/Views/HighLevel/Requests/RequestImplementations.cshtml

```diff
@@ -5,14 +5,32 @@
 @using ApiGenerator.Domain
 @using ApiGenerator.Domain.Code
 @using ApiGenerator.Domain.Code.HighLevel.Requests
+@using ApiGenerator.Domain.Specification
 @using ApiGenerator.Generator
+@using CsQuery.ExtensionMethods.Internal
 @inherits global::ApiGenerator.CodeTemplatePage<RequestPartialImplementation>
 @{
 	RequestPartialImplementation r = Model;
 	CsharpNames names = r.CsharpNames;
 	var apiLookup = $"ApiUrlsLookups.{r.CsharpNames.Namespace}{r.CsharpNames.MethodName}";
 }
-	///<summary>Request for @names.MethodName <para>@r.OfficialDocumentationLink</para></summary>
+///<summary>Request for @names.MethodName@Raw(r.OfficialDocumentationLink.IsNullOrEmpty() ? "" : " <para>" + r.OfficialDocumentationLink + "</para>")</summary>
+@if (r.Stability != Stability.Stable)
+{
+	string warningMessage = "";
+	switch (r.Stability)
+	{
+		case Stability.Experimental:
+			warningMessage = "it is highly likely to break or be removed in the near future and has no backwards compatibility guarantees.";
+			break;
+		case Stability.Beta:
+			warningMessage = "it is less likely to break or be removed but still reserve the right to do so.";
```

Comment:
I think we should use the same wording as the reference documentation for _beta_:

> This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.